### PR TITLE
add var for remote system ip in lldp tpl

### PR DESCRIPTION
**Pull request by ratneshnagori**
Current textfsm ios template for show lldp neighbors details doesnt return remote system's mgmt ip. Editing this template to parse output of command to include mgmt ip as well.

### CODE REVIEW

File: napalm/ios/utils/textfsm_templates/show_lldp_neighbors_detail.tpl [hunk 1]
```diff
@@ -17,7 +17,8 @@ Start
   ^System Description: -> Description
   ^System Description\s*-\s*${REMOTE_SYSTEM_DESCRIPTION}
   ^System Capabilities\s*?[:-]\s+${REMOTE_SYSTEM_CAPAB}
-  ^Enabled Capabilities\s*?[:-]\s+${REMOTE_SYSTEM_ENABLE_CAPAB} -> Record
+  ^Enabled Capabilities\s*?[:-]\s+${REMOTE_SYSTEM_ENABLE_CAPAB}
+  ^\s+IP\:\s${REMPRT_SYSTEM_MGMT_IP} -> Record
```
Comment (mirceaulinic) [inline napalm/ios/utils/textfsm_templates/show_lldp_neighbors_detail.tpl:21]: What is remprt and where is this variable used?

Comment (ratneshnagori) [reply napalm/ios/utils/textfsm_templates/show_lldp_neighbors_detail.tpl:21]: Actually correct name is REMOTE_SYSTEM_MGMT_IP. I will correct.

This variable is being used to get remote system's management ip which is used to build a network topology. Once you have remote system's management ip, then you can log into it and find devices downstream.

**Review by mirceaulinic — CHANGES_REQUESTED** [review]
The tests are failing: https://github.com/napalm-automation/napalm/runs/6521163875?check_suite_focus=true#step:8:1199

```
Wrong format of TextFSM template show_lldp_neighbors_detail: Expected blank line after last Value entry. Line: 10.
```

**Comment by ratneshnagori** [comment]
> The tests are failing: https://github.com/napalm-automation/napalm/runs/6521163875?check_suite_focus=true#step:8:1199
> 
> ```
> Wrong format of TextFSM template show_lldp_neighbors_detail: Expected blank line after last Value entry. Line: 10.
> ```

Hi. Added a new line after var declaration - https://github.com/ratneshnagori/napalm/commit/b5344e7582b6d35b81a6fac86ecde180c341ee27

**Comment by mirceaulinic** [comment]
@ratneshnagori I'm afraid the tests are still failing...

**Pull request closed without merging**